.github/workflows/test-rust.yaml

```diff
@@ -66,18 +75,13 @@ jobs:
         with:
           resource: "tcp:5432 tcp:3306 tcp:1433"
           timeout: 60000
-        if: ${{ inputs.target== 'x86_64-unknown-linux-gnu' }}
-        # We previously split up the test compilation as recommended in
-        # https://matklad.github.io/2021/09/04/fast-rust-builds.html, but the
-        # binary is different for `cargo insta test`, so it ends up compiling
-        # twice.
```

Comment:
Is this change correct?
I believe this prerequisite is necessary because it seems that if the Docker container does not exist it will not succeed.

Reply:
Sorry! The comment is correct to remove; the `if` is wrong